packages/features/bookings/lib/handleSeats/reschedule/attendee/attendeeRescheduleSeatedBooking.ts

```diff
@@ -76,17 +77,19 @@ const attendeeRescheduleSeatedBooking = async (
     ]);
   }
 
-  const copyEvent = cloneDeep(evt);
-
-  const updateManager = await eventManager.reschedule(copyEvent, rescheduleUid, newTimeSlotBooking.id);
-
-  const results = updateManager.results;
+  // Add the new attendees to the new time slot booking attendees
+  for (const attendee of newTimeSlotBooking.attendees) {
+    const language = await getTranslation(attendee.locale ?? "en", "common");
+    evt.attendees.push({
+      email: attendee.email,
+      name: attendee.name,
+      language,
+    });
+  }
 
-  const calendarResult = results.find((result) => result.type.includes("_calendar"));
+  const copyEvent = cloneDeep({ ...evt, iCalUID: newTimeSlotBooking.iCalUID });
 
-  evt.iCalUID = Array.isArray(calendarResult?.updatedEvent)
-    ? calendarResult?.updatedEvent[0]?.iCalUID
-    : calendarResult?.updatedEvent?.iCalUID || undefined;
+  await eventManager.updateCalendarAttendees(copyEvent, newTimeSlotBooking);
```

Comment:
Only update the new time slot calendar event with the rescheduled attendee. Emails are still being sent to the organizer and rescheduled attendee.